### PR TITLE
Add check for manual operative assignment after individual GET

### PR DESCRIPTION
**Pull request by Asura5**

### CODE REVIEW

File: RepairsApi/V2/UseCase/SyncDRSWorkOrdersUseCase.cs [hunk 1]
```diff
@@ -148,6 +148,12 @@ public async Task<bool> ValidateWorkOrder(WorkOrder wo)
         {
             const int PlannedPriorityCode = 9;
 
+            if (wo.HasManuallyAssignedOperatives())
```
Comment (LBHCallumM) [inline RepairsApi/V2/UseCase/SyncDRSWorkOrdersUseCase.cs:151]: Pretty sure this line is being called inside `GetWOrkOrdersFromToday` and `GetHistoricalWorkOrders`.

**Review by LBHCallumM — APPROVED** [review]
Approved, except for the other comment

File: RepairsApi/V2/UseCase/SyncDRSWorkOrdersUseCase.cs (hunk 1 above)
Comment (LBHCallumM) [reply RepairsApi/V2/UseCase/SyncDRSWorkOrdersUseCase.cs:151]: sure

Comment (Asura5) [reply RepairsApi/V2/UseCase/SyncDRSWorkOrdersUseCase.cs:151]: join drop-in

**Pull request merged**